users/views.py

```diff
@@ -1,10 +1,13 @@
 import json, bcrypt, jwt
+from django.core.exceptions import ValidationError
 
 from django.views import View
 from django.http  import JsonResponse, HttpResponse
 
 from .models      import User
+
 from my_settings  import SECRET_KEY, ALGORITHM
+
 from .validation  import validate_email, validate_phone_number, validate_password
 from core.utils   import login_decorator
```

Comment:
불필요하게 너무 많은 공백이 더해진 것 같습니다! 수정해주세요~
```suggestion
import json, bcrypt, jwt

from django.core.exceptions import ValidationError
from django.views           import View
from django.http            import JsonResponse, HttpResponse

from .models      import User
from my_settings  import SECRET_KEY, ALGORITHM
from .validation  import validate_email, validate_phone_number, validate_password
from core.utils   import login_decorator
```

Reply:
수정하였습니다!